invenio_oauthclient/alembic/97bbc733896c_create_oauthclient_tables.py

```diff
@@ -73,6 +73,7 @@ def upgrade():
         sa.Column('secret', sa.Text(), nullable=False),
         sa.ForeignKeyConstraint(
             ['id_remote_account'], [u'oauthclient_remoteaccount.id'],
+            name='fk_oauthclient_remote_token_remote_account'
```

Comment:
The issue here is that on MySQL databases, which support constraint names of up to 60 characters, this alembic recipe will fail anyway (as the name is longer), so the migration will fail before getting to the other recipe.